src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -156,7 +304,13 @@ private static void SetPreciseMachineCounters(IEnumerable<ProviderInfo> provider
                 }
 
                 if (profileSourceIDs.Count > 0)
+                {
+                    // 
+                    // FIXME: This function changes the -pmcsources intervals machine wide.
```

Comment:
Ok, if not a new issue, can you please file an issue for this?

---
In reply to: [102335560](https://github.com/Microsoft/xunit-performance/pull/191#discussion_r102335560) [](ancestors = 102335560)